src/plugins/chart_expressions/expression_xy/common/i18n/index.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const strings = {
```

Comment:
```suggestion
export const helpText = {
```
nit: I am not sure of this name... Especially because we export it to another file. What about `helpText`?

Reply:
This pattern has been used in a lot of other plugins, related to the expressions. What about supporting consistency? I'm not insisting, just asking 😃 
<img width="544" alt="Screenshot 2022-04-19 at 13 17 27" src="https://user-images.githubusercontent.com/22456368/163982837-8e2f2f14-2f32-4270-b159-f0bdc6e7e042.png">

Reply:
Ah ok, didn't know that. I guess it's fine :D